girder/api/v1/file.py

```diff
@@ -109,16 +109,38 @@ def initUpload(self, parentType, parentId, name, size, mimeType, linkUrl, refere
                 self.requireAdmin(
                     user, message='You must be an admin to select a destination assetstore.')
                 assetstore = self.model('assetstore').load(assetstoreId)
+
+            chunk = None
+            if 'chunk' in params:
+                chunk = params['chunk']
+                if isinstance(chunk, cherrypy._cpreqbody.Part):
```

Comment:
Hm, you're right -- I thought we had documented this preference somewhere, but I can't find anything. I don't think we've deprecated it per se since we probably want to keep it around for a while for very old clients, but we definitely need to officially discourage it. I'll open a ticket.